src/nvim/eval.c

```diff
@@ -22210,6 +22208,19 @@ static bool func_remove(ufunc_T *fp)
   return false;
 }
 
+static void func_clear_items(ufunc_T *fp)
+{
+  ga_clear_strings(&(fp->uf_args));
+  ga_clear_strings(&(fp->uf_lines));
+
+  xfree(fp->uf_tml_count);
+  fp->uf_tml_count = NULL;
+  xfree(fp->uf_tml_total);
+  fp->uf_tml_total = NULL;
+  xfree(fp->uf_tml_self);
+  fp->uf_tml_self = NULL;
```

Comment:
Why not use `XFREE_CLEAR`?

Reply:
Good point.
I wasn't aware of it.
But I guess this makes the diff to Vim a bit better (although using `xfree` instead already, of course).

Reply:
Created https://github.com/neovim/neovim/pull/10436.